imports/api/playlists/server/publications.js

```diff
@@ -16,6 +16,17 @@ Meteor.publish('playlistsLimited', function(options) {
   return Playlists.find({}, options);
 });
 
+Meteor.publish('currentPlaylist', function() {
+  return Playlists.find({$where: function() {
+    return this.showDate.getYear() === new Date().getYear() &&
```

Comment:
I'm seeing this function a few times. Can we factor it out into a separate function elsewhere in the project? This will improve maintainability.